rootfs/etc/templates/haproxy/haproxy.tmpl

```diff
@@ -789,6 +789,7 @@ backend {{ $backend.ID }}
         {{- "" }} weight {{ $ep.Weight }}
         {{- if and ($backend.CookieAffinity) ($ep.CookieValue) }} cookie {{ $ep.CookieValue }}{{ end }}
         {{- if $ep.SourceIP }} source {{ $ep.SourceIP }}{{ end }}
+        {{- if ne (int64 $ep.PUID) 0 }} id {{ $ep.PUID }}{{ end }}
```

Comment:
this can be simplified to `{{- if $ep.PUID }} id {{ $ep.PUID }}{{ end }}`

Reply:
OK. I don't know this template language, so I was copying from a nearby example :)

Reply:
Done.